cogs/welcome/welcome.py

```diff
@@ -51,19 +53,18 @@ def __init__(self, bot: Red):
         self.config.register_guild(**DEFAULT_GUILD)
 
     async def getRandomMessage(self, guild):
-        greetings = await self.config.guild(guild).KEY_GREETINGS()
+        greetings = await self.config.guild(guild).greetings()
         numGreetings = len(greetings)
-        if numGreetings == 0:
+        if not greetings:
             return "Welcome to the server {USER}"
         else:
-            return greetings[random.randint(0, numGreetings - 1)][1]
+            return random.choice(list(greetings.values()))
```

Comment:
greetings.values() should already be a list:
```suggestion
            return random.choice(greetings.values())
```

Reply:
Have to use list(), I tried and it doesn't work without it as it's not a list, it's a value thingy or something. I don't understand it

Reply:
Yeah sorry, you're right, was thinking in python2 terms earlier.